src/platforms/ruby/config.yml

```diff
@@ -1,3 +1,4 @@
 title: Ruby
 caseStyle: snake_case
 supportLevel: production
+sdk: "sentry.ruby"
```

Comment:
FYI - there's something odd with the Getting Started page, as it's missing the line from the common content that introduces the frameworks.